problemtools/problem2html.py

```diff
@@ -7,14 +7,15 @@
 import logging
 import subprocess
 
-import plasTeX.TeX
-import plasTeX.Logging
-
-from .ProblemPlasTeX import ProblemRenderer
-from .ProblemPlasTeX import ProblemsetMacros
-from . import template
 
 def convert(options: argparse.Namespace) -> None:
+    import plasTeX.TeX
```

Comment:
Unless we have a strong reason not to, we try to follow PEP8. In PEP8 imports should be at the top of the file.

Reply:
Hmm right I should have moved that to a separate issue. The intent was that verifyproblem then runs without plastex installed.

But I don't have a strong opinion on this, can revert this.

Reply:
fine with me. But then let's be honest about the plastex dependency and add it to requirements.txt :-) (added)

Reply:
Nvm, it is in setup.py!